std/experimental/allocator/package.d

```diff
@@ -3113,3 +3139,4 @@ unittest
     b = alloc2.allocate(101);
     assert(alloc2.deallocate(b));
 }
+
```

Comment:
Unrelated?

Reply:
Yes, things were initially at the end, until i realize that because of the `__EOF__` the test were not executed and the ddoc not rendered.